CHANGELOG.md

```diff
@@ -33,6 +33,11 @@ A brief description of the categories of changes:
 ### Removed
 * Nothing yet
 
+### Fixed
+* (bzlmod) get the path to the host python interpreter without calling
+  `mctx.path` on Labels that have differing contents on different OSes.
+* (bzlmod) correctly watch sources when using `pypi_repo_utils`.
```

Comment:
Can you rephrase this so it isn't using internal jargon? "pypi_repo_utils" is an internal thing not relevant to users.